chain-impl-mockchain/src/txbuilder.rs

```diff
@@ -289,3 +289,153 @@ impl TransactionFinalizer {
         }
     }
 }
+
+#[cfg(test)]
```

Comment:
consider making this test as public interface test (by moving them to `rust-cardano/tests`). Thanks to that we can split its logic across separate files.

Reply:
Do you mean the whole test module or just the helper functions?

Reply:
I meant the whole test module,

Reply:
I'm not sure about that. This test touches only TransactionBuilder and nothing else, it's more of a unit test than integration test. If we keep it close to TransactionBuilder, it will be much easier to find.

Reply:
sure, this is good reason